src/ui/components/pages/Catalog/CatalogExplorer/CatalogExplorerCards/CatalogExplorerCards.tsx

```diff
@@ -21,14 +21,19 @@ export type Props = {
     className?: string;
     packages: {
         packageName: string;
+        version: string;
```

Comment:
There is no reason to pass the version number here. It's not displayed anywhere.